src/TodoAction.jsx

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+
+export default function TodoAction({ todoAction, index, onTodoActionToDone }) {
+  const handleTodoActionToDone = (event) => {
+    const { value } = event.target;
+    const listIndex = Number(value);
+    onTodoActionToDone(listIndex);
+  };
+
+  return (
+    <div>
+      { todoAction }
+      <button type="button" onClick={handleTodoActionToDone} value={index}>완료</button>
+    </div>
+  );
```

Comment:
```suggestion
  const handleClick = () => {
    onTodoActionToDone(index);
  };

  return (
    <div>
      { todoAction }
      <button type="button" onClick={handleClick}>완료</button>
    </div>
  );
```

DOM의 기능보다는 이렇게 사용하는게 더 좋을 것 같아요. 왜냐하면 이미 `index`를 알고 있으니, DOM에서 다시 읽을 필요 없을 것 같아요